Source/WebCore/dom/FullscreenManager.h

```diff
@@ -108,7 +107,10 @@ class FullscreenManager final : public CanMakeWeakPtr<FullscreenManager> {
     WTFLogChannel& logChannel() const;
 #endif
 
+    Document& topDocument() { return m_topDocument ? *m_topDocument : document().topDocument(); }
```

Comment:
I think this would look clearer if this was:
`Document& topDocument() { return m_topDocument ? *m_topDocument : m_document; }`

It was like this in an earlier iteration I think. Not sure why you made this change (I don't think it should be needed).